spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -252,8 +253,13 @@ def getHeader(productName, options, orgCACert, isRpmYN, pubname, apachePubDirect
     # Removing paths from options.gpg_key
     org_gpg_key = ",".join([os.path.basename(gpg_key) for gpg_key in options.gpg_key.split(",")])
     exit_call = " " if options.activation_keys or options.salt else "exit 1"
+    with cfg_component('web') as CFG:
+        version = CFG.version
+        if isUyuni:
+            version = CFG.uyuni
```

Comment:
This logic is wrong here as you are not calling `isUyuni()` but just checking it exists. 

I think what you need is:

```suggestion
        version = CFG.uyuni if isUyuni() else CFG.version
```

Reply:
The "ugly" bit here (unrelated to you suggestion, that's toally valid!) is that `CFG.uyuni` is defined as `CFG.version.uyuni` in `rhn_web.conf` but accessed as `CFG.uyuni`.